extensions/blocks/google-calendar/edit.js

```diff
@@ -109,7 +109,7 @@ class GoogleCalendarEdit extends Component {
 	 */
 	render() {
 		const { attributes, className } = this.props;
-		const primaryClass = className.split( ' ', 1 );
+		const primaryClass = className ? className.split( ' ', 1 )[ 0 ] : '';
```

Comment:
This would definitely work. Under the hood, the `className` is created using `getBlockDefaultClassName`, which is what I opted to use in #14725. That would make this something like:

```
const primaryClass = getBlockDefaultClassName( this.props.name );
```

Reply:
Haha - your version is much less hacky - have switched to that :-)